src/screens/ContactScreen/index.js

```diff
@@ -18,6 +18,7 @@ import {Loading} from '../../components';
 import {Search} from './elements';
 import {userPopulate} from '../../service/users';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {ANONYMOUS} from '../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ANONYMOUS` from `../../hooks/core/constant` is a new addition. This constant seems to be used for checking the source screen and handling different scenarios based on it. Using constants for such checks improves code readability and maintainability.
```diff
+ import {ANONYMOUS} from '../../hooks/core/constant';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->